packages/shared/components/atoms/pills/ActivityTypePill.svelte

```diff
@@ -0,0 +1,68 @@
+<script lang="typescript">
```

Comment:
Why did you go for the name activity? Transaction seems more appropriate, since it's part of what we refer to as Transaction History.

Reply:
Because some of the components have always been called activity, and in Stardust upgrade I am trying to future proof this for when the wallet activity panel might contain other things that the transaction history.